org.gimp.GIMP.json

```diff
@@ -152,8 +152,8 @@
             "sources" : [
                 {
                     "type": "archive",
-                    "url": "https://github.com/AcademySoftwareFoundation/openexr/archive/refs/tags/v3.3.1.tar.gz",
-                    "sha256": "58aad2b32c047070a52f1205b309bdae007442e0f983120e4ff57551eb6f10f1",
+                    "url": "https://github.com/AcademySoftwareFoundation/openexr/archive/refs/tags/v3.3.2.tar.gz",
+                    "sha256": "20e40baa6654785d69642e6e85542968db3c5d08da630adc590ff066a52c5938f4ce8a77c0097e00010a905c8c31d8f131eb0308a3f8b6439ab6be4133eae246",
```

Comment:
this is not a sha256. it's way too long.

Reply:
Clunky copy paste error of mine